packages/web-components/fast-foundation/src/select/select.spec.ts

```diff
@@ -6,7 +6,7 @@ import { Select, selectTemplate as template } from "./index";
 import { KeyCodes } from "@microsoft/fast-web-utilities";
 
 const FASTSelect = Select.compose({
```

Comment:
looks like this needs casing fixes as well

Reply:
Ahhh - that's all the tests. @EisenbergEffect are you good if I do a test pass to update the casing for the internal instances?

Reply:
Oh. I didn't realize that. Definitely not as important in the tests. That's all internal. We can clean that up whenever we have time.